introduction/de/06-conclusion.asciidoc

```diff
@@ -0,0 +1,13 @@
+== Schusswort
+
+Auf dieser Serie haben wir eine Einführung in InnerSource gegeben.
+InnerSource wendet Open Source Praktiken und Prinzipien auf die firmeninterne Softwareentwicklung an.
+Es bietet Nutzern eine zusätzliche Option, speziell wenn Produktionsteams nicht in der Lage sind eine erforderliche Funktionsanforderung im notwendingen Zeitraum zu liefern.
+Erfolgreiche InnerSource umfasst https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/product-owner/01-opening-article.asciidoc[_Produkt Owner_] und https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-Committer/01-Introduction.asciidoc [_Trusted Committer_] vom *Hostteam* sowie ein https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc [_Contributor_] vom *Gastteam*.
+Richtig betriebene InnerSource bietet beiden teilnehmenden Teams viele Vorteile.
+Die Schlüsselprinzipien, nach denen gut geführte InnerSource funktioniert, sind *freiwilliger Code-Beitrag* und *priorisierte Schulung/Betreuung*.
```

Comment:
```suggestion
Die Schlüsselprinzipien, nach denen gut geführte InnerSource Projekte funktionieren, sind *freiwilliger Code-Beitrag* und *Mentoring Fokus*.
```

Reply:
I think leaving the "prioritized" thing here is important, as delayed interaction, or even deprioritization of contributions, especially first ones, to bulk rate is very efficient at killing contributions. 
Any not exactly uncommon.